frontend/src/scenes/funnels/funnelLogic.js

```diff
@@ -2,7 +2,30 @@ import { kea } from 'kea'
 import api from 'lib/api'
 import { toast } from 'react-toastify'
 import { ViewType, insightLogic } from 'scenes/insights/insightLogic'
-import { objectsEqual } from 'lib/utils'
+import { objectsEqual, toParams } from 'lib/utils'
+
+function wait(ms = 1000) {
+    return new Promise((resolve) => {
+        setTimeout(resolve, ms)
+    })
+}
+
+const SECONDS_TO_POLL = 120
```

Comment:
Open for suggestion here: How long should we poll for and how should we handle timing out? Just have a popup toast that suggests a retry?